### PR TITLE
Fix FeeBumpTransaction submit.

### PR DESCRIPTION
**Pull request by Kirbyrawr**
- Added FeeBumpTransaction in SubmitTransactionOptions.
- Now SubmitTransaction checks if it's FeeBump or not.
- Added FromEnvelopeXdr in FeeBumpTransaction.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] I have read the **CONTRIBUTING** document.
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: stellar-dotnet-sdk/Server.cs [hunk 1]
```diff
@@ -93,7 +92,7 @@ public RootResponse Root()
         /// <returns></returns>
         public Task<SubmitTransactionResponse> SubmitTransaction(Transaction transaction)
```
Comment (fracek) [inline stellar-dotnet-sdk/Server.cs:93]: We need two overloads of `SubmitTransaction` that accept a `FeeBumpTransaction`.

* `SubmitTransaction(FeeBumpTransaction tx)`
* `SubmitTransaction(FeeBumpTransaction tx, SubmitTransactionOptions options)`

These overloads should set the options for feebumptransaction to true.

**Review by fracek — CHANGES_REQUESTED** [review]
Looks good but we should add the two overloads i mentioned.

File: stellar-dotnet-sdk/Server.cs (hunk 1 above)
Comment (Kirbyrawr) [reply stellar-dotnet-sdk/Server.cs:93]: Done

**Review by fracek — APPROVED** [review]
Thank you for the changes!

**Pull request merged**